doc/source/getting_started/XarrayforArviZ.ipynb

```diff
@@ -12,17 +12,17 @@
    "cell_type": "markdown",
```

Comment:
<p>I think we should use netCDF as a general rule but treat it as a word so the <code>N</code>  is capitalized at the start of a sentence and so on. </p><p> </p><p>The manual anchor from which to link in the quickstart should be right above the netcdf title, with the <code>(anchor_id)=</code>  format.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>